src/ddmd/parse.d

```diff
@@ -21,6 +21,7 @@ import ddmd.root.filename;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 import ddmd.root.rootobject;
+import ddmd.target;
```

Comment:
Unfortunately, adding this line brings in the rest of the compiler, including the backend, as a dependency on this module. The parser was separate from most of the rest of the compiler in this PR: https://github.com/dlang/dmd/pull/6625.